### ISSUE TITLE
Why the accuracy and speed of retrained model is not as good as the sst300_0712_83000.pth which is given by author？

### ISSUE THREAD

**Issue by hanshong** [shijieS/SST#51]
For training on MOT17, I have not modified any parameters except batch_size=2 and num_workers=0. And I am not sure whether the learning rate(1e-6) is too low that the loss is non-convergent and always between 1.4-2.6 in the training process. In the generated images, the connection lines between the front and back frame's objects disappears, or all lines points to the same object in the back frame.
For testing on MOT17, the predict trajectories are scattered and wrong straight lines while sst300_0712_83000 can generate great and precise trajectories.

Are there some parameters that need to be adjusted when retraining on MOT17?  Hope for response, thanks.
